src/action/github.ts

```diff
@@ -0,0 +1,30 @@
+import {PathLike} from "fs";
+import {Action} from "../action";
+import core from "@actions/core";
+import {Output} from "../config/output";
+
+export class Github implements Action {
+    debug(message: string): void {
+        core.debug(message)
+    }
+
+    info(message: string): void {
+        core.info(message)
+    }
+
+    error(message: string): void {
```

Comment:
I've removed the interface extension and instead added a method `Action#getLogger`. This will actually return the logger so that can be injected and used instead.